README.md

```diff
@@ -147,4 +155,4 @@ to the process (which results in an immediate termination, and this signal
 can't be caught).
 
 [badgepub]: https://circleci.com/gh/scality/Arsenal.svg?style=svg
```

Comment:
this badge is not relevant anymore: we are not using circleci anymore, but only github actions.

c.f. https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge